webpack/webpack.dev.js

```diff
@@ -0,0 +1,10 @@
+const { merge } = require('webpack-merge');
+const common = require('./webpack.common.js');
+
+module.exports = merge(common, {
+  mode: 'development',
+  devtool: 'inline-source-map',
+  devServer: {
+    contentBase: './dist'
```

Comment:
Help me change contentBase to `./build` cause we will use `dist` for entry point of public package.

Reply:
Done!